librz/include/rz_core.h

```diff
@@ -484,8 +484,8 @@ RZ_API RzLineNSCompletionResult *rz_core_autocomplete_rzshell(RzCore *core, RzLi
 RZ_IPI void rz_core_print_scrollbar(RzCore *core);
 RZ_IPI void rz_core_print_scrollbar_bottom(RzCore *core);
```

Comment:
move these IPI within the private header

Reply:
That is unrelated, better make another pr for it.